src/console.cpp

```diff
@@ -552,8 +552,8 @@ static void setup_commands(std::shared_ptr<Commands> & commands) {
 
             if (return_code == CommandRet::OK) {
                 if (json.size()) {
-                    if (json.containsKey("api_data")) {
-                        String data = json["api_data"].as<String>();
+                    if (json["api_data"].is<String>()) {
```

Comment:
better std::string

Reply:
thanks - changed to std::string to keep consistent